aea/cli/run.py

```diff
@@ -118,26 +119,23 @@ def run(
     if exclude_connection_ids:
         connection_ids = _calculate_connection_ids(ctx, exclude_connection_ids)
 
-    if profiling > 0:
-        with _profiling_context(period=profiling):
-            run_aea(
-                ctx,
-                connection_ids,
-                env_file,
-                is_install_deps,
-                apply_environment_variables,
-                password,
-            )
-            return
-    run_aea(
-        ctx,
-        connection_ids,
-        env_file,
-        is_install_deps,
-        apply_environment_variables,
-        password,
+    profiling_context = (
+        _profiling_context(period=profiling) if profiling > 0 else nullcontext()
+    )
+    memray_context = (
+        memray.Tracker("memray_profiling.bin") if profiling > 0 else nullcontext()
```

Comment:
Can we use the `memory_interval_ms` parameter to reduce the growth rate? Try it if you haven't yet, I'm not sure it will because documentation is vague (last part of sentence: https://bloomberg.github.io/memray/api.html).

Reply:
According to that sentence: `this parameter lets you adjust the frequency between updates, though you shouldn’t need to change it`. What is the motivation to use a different one then?